scripts/xe-enable-ipv6

```diff
@@ -32,10 +32,8 @@ if [ $1 = "enable" ]; then
 	echo "IPV6_AUTOCONF=NO" >> /etc/sysconfig/network
 	chkconfig ip6tables on
 
-	sed -i '/^net.ipv6.conf.all.disable_ipv6=/d' /etc/sysctl.d/91-net-ipv6.conf
-	sed -i '/^net.ipv6.conf.default.disable_ipv6=/d' /etc/sysctl.d/91-net-ipv6.conf
-	echo "net.ipv6.conf.all.disable_ipv6=0" >> /etc/sysctl.d/91-net-ipv6.conf
-	echo "net.ipv6.conf.default.disable_ipv6=0" >> /etc/sysctl.d/91-net-ipv6.conf
+	echo "net.ipv6.conf.all.disable_ipv6=0" > /etc/sysctl.d/91-net-ipv6.conf
+	echo "net.ipv6.conf.default.disable_ipv6=0" > /etc/sysctl.d/91-net-ipv6.conf
```

Comment:
```suggestion
	echo "net.ipv6.conf.default.disable_ipv6=0" >> /etc/sysctl.d/91-net-ipv6.conf
```

Reply:
Both lines have to be added ;)

Reply:
Yeah I saw that, not fully awake this morning i guess ^^'